packages/flutter_lints/README.md

```diff
@@ -58,7 +58,21 @@ linter:
 # https://dart.dev/guides/language/analysis-options
 ```
 
+## Adding new lints
+
+Please file a [lint proposal] issue to suggest that an existing lint rule should
```

Comment:
I think you need `[lint proposal][]` for that link to work?

Reply:
The style without the `[]` also works: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

I also verified that the links further up in the readme using the same style link correctly, see https://pub.dev/packages/flutter_lints/versions/2.0.0.